src/prezzemolo/vertex.py

```diff
@@ -56,6 +56,18 @@ def __eq__(self, other: object) -> bool:
             raise TypeError(f"Operand has non-Vertex value: {repr(other)}")
         return self.name == other.name
 
+    def __lt__(self, other: object) -> bool:
+        if not other:
+            return False
+        if not isinstance(other, Vertex):
+            raise TypeError(f"Operand has non-Vertex value: {repr(other)}")
+        setlocale(LC_ALL, "en_US.utf8")
```

Comment:
There are a couple of problems with using `setlocale()` here:
* `setlocale()` is typically not called in libraries, but only in app code 
* `setlocale()` should not be called multiple times with the same value, but only once at the beginning of the app (it can be called later too if the app wants to change locale).

What happens if we don't set the locale and we just do a normal string comparison?

Reply:
> To summarize, if you remove the locale code from vertex.py, I think we can merge this. The string comparison can simply be:
> 
> ```
> return self.name < other.name
> ```

It's all right if capital names appear before lowercase ones [because > < sort strings according to their ASCII value](https://pythonexamples.org/python-greater-than/#4)? ie.

- Apple
- Banana
- apple
- banana

I thought that wasn't intuitive. That is why I was using `strcoll(self.name, other.name)` I wanted to lock the locale because if you don't I think it will use the system default, which would be different for different regions, potentially invalidating the tests.

Reply:
Yes, in ASCII capital letters are before lowercase ones. Locale cannot be set by a library, because if all libraries set the locale, then it would be impossible to understand what locale the app would be using. This is why locale setting is left to the app. Our tests only use ASCII strings with character codes < 127, so I don't think we'll have any problem.